R/text_definition.R

```diff
@@ -0,0 +1,11 @@
+text_definition <- function(key, lang = "en", glossary = NULL) {
+ if (is.null(glossary)){
+    glossary <- get_glossary()
+  }
+
+  entries <- glossary$rmd_define(key = key, lang = lang)
+  entries_list <- purrr::map(entries, ~.x$.__enclos_env__$private$.entries)
```

Comment:
Should we just add a public method to the object for grabbing the entries list instead of relying on this pattern?

Reply:
I tried... and couldn't figure it out. If you can help me adding that method for the next version I would be grateful.

Reply:
Would you like me to add this to the current PR?

Reply:
Let's leave it for the next PR and we will put it as a priority

Reply:
Added in Issue #13